README.md

```diff
@@ -9,6 +9,41 @@ HPCC Systems offers an enterprise ready, open source supercomputing platform to
 * [Installation and Running](https://hpccsystems.com/training/documentation/installation-and-administration)
 * [Developer Documentation](https://hpcc-systems.github.io/HPCC-Platform/)
 
+# Release + Support Policy
+
+In general, a new version of the HPCC Platform is released every 6 months. These releases can be either Major (with breaking changes) or Minor (with new features). Maintenance and security releases (point releases) are typically made weekly, and may occasionally include technical previews.
+
+Maintenance releases are supported for the current and previous release, while security releases are supported for the current and previous two releases:
+
+```mermaid
+---
+displayMode: compact
+---
+gantt
+    title Release Schedule
+    axisFormat %Y-Q%q
+    tickInterval 3month
+    dateFormat YYYY-MM-DD
+    section v8.12.x
+        Active:          done, 2023-02-07, 6M
+        Security:        done, 9M
+    section v9.0.x
+        Active:          done, 2023-04-01, 9M
+        Security:        active, 6M
+    section v9.2.x
+        Active:          active, 2023-07-01, 6M
+        Security:        6M
+    section v9.4.x
+        Active:          active, 2023-10-01, 9M
+        Security:        6M
+    section v9.6.x
+        Active:          2024-04-01, 1y
```

Comment:
, 6M and also line 43.  Standard is really 6M active, 6M security, but it has changed a bit around the 9.4.x release.